tool/menu.lua

```diff
@@ -148,36 +148,55 @@ end
 
 local function randomizermenu()
   local selected = ui.select({
+    "New Seed on New Game    " .. randomizer.newgamenewseedui[randomizer.newgamenewseed],
     "Proficiency: " .. randomizer.proficiency,
-    "Unseeded",
-    "Seeded",
+    "Random seed",
+    "Set seed",
     "Reset",
     "Back",
   })
-  if selected == 1 then
+  if selected == 1 then -- New Seed on New Game
+    local index = table.indexof(randomizer.newgamenewseedvalues, randomizer.newgamenewseed)
+    index = ((index - 1 + 1) % #randomizer.newgamenewseedvalues) + 1
+    randomizer.newgamenewseed = randomizer.newgamenewseedvalues[index]
+
```

Comment:
I don't see the need for newlines before `elseif`, but idm either way.

Reply:
Adding these and the comments made it easier for me to navigate the code, if you don't mind I'd prefer to keep them.